### DISCUSSION TITLE
Hedef ortaklık düşünceniz var mı?

### DISCUSSION THREAD

**Discussion by elonunmk** [IBAX-io/go-ibax#1595] [Soru Cevap - Türkçe]
IBAX.io olarak yakın, orta ve uzun vadede düşündüğünüz ve hatta iletişimde olduğunuz herhangi bir proje ile ortaklık düşünceniz var mı?

**Comment by Yusufknk** [comment]
Şuanlık coinmarketcap, Coingecko ve büyük hacimli borsalarla görüşme halindeler netleşen detaylar sizlere duyurulacaktır.

**Reply by elonunmk** [reply]
Bilgilendirme için teşekkür ederim. Takipteyim.
